dojo/api.py

```diff
@@ -1457,8 +1458,8 @@ def obj_create(self, bundle, **kwargs):
         try:
             t.full_clean()
         except ValidationError as e:
-            print "Error Validating Test Object"
-            print e
+            print("Error Validating Test Object")
```

Comment:
What about?

```suggestion
            print("Error Validating Test Object\n ".format(e))
```

Reply:
That would halt the program.  The goal of this PR is not to optimize or improve the code.  We want to make safe changes to get it closer to Python 3 compatibility.